client/pages/connectpage/form.js

```diff
@@ -116,7 +116,7 @@ export class Form extends React.Component {
                  </div>
               </NgIf>
               <div>
-                <form onSubmit={this.onSubmit.bind(this)} autoComplete="off" autoCapitalize="off" spellCheck="false" autoCorrect="off">
+                <form onSubmit={this.onSubmit.bind(this)} autoComplete="on" autoCapitalize="off" spellCheck="false" autoCorrect="off">
```

Comment:
Some phone have weird behaviors when it comes to the autocomplete and I'd rather give hints to the browser to disable them instead of doing it wrong